### DISCUSSION TITLE
Unable to build on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by amansharma612** [openvinotoolkit/openvino#27663] [General]
Hi, I am encountering the following error when building openvino from source.

`/usr/bin/ld: platform.cpp:(.text._ZN13MLAS_PLATFORMC2Ev+0xf2): undefined reference to 'MlasConvNchwcFloatKernelFma3'
/usr/bin/ld: platform.cpp:(.text._ZN13MLAS_PLATFORMC2Ev+0x102): undefined reference to 'MlasConvDepthwiseFloatKernelFma3'
/usr/bin/ld: platform.cpp:(.text._ZN13MLAS_PLATFORMC2Ev+0x129): undefined reference to 'MlasConvPointwiseFloatKernelFma3'
/usr/bin/ld: platform.cpp:(.text._ZN13MLAS_PLATFORMC2Ev+0x3c6): undefined reference to 'MlasGemmFloatKernelFma3'`

can anyone suggest a fix for the same?

**Comment by azhai219** [comment]
@amansharma612 can you share the build command and the CPU? Thanks.
